swap_meet/vendor.py

```diff
@@ -1,2 +1,69 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
```

Comment:
This works just fine! Consider doing a little research on ternary expressions. The syntax is similar to list comprehension. By using a ternary operator, we can get rid of the noise inside the constructor method.
```python
self.inventory = inventory if inventory is not None else []
```
Some extra information on [ternary expressions](https://www.geeksforgeeks.org/ternary-operator-in-python/)